osu.Game/Screens/OnlinePlay/DailyChallenge/DailyChallengeScoreBreakdown.cs

```diff
@@ -70,34 +68,61 @@ protected override void LoadComplete()
             });
         }
 
+        private readonly Queue<NewScoreEvent> newScores = new Queue<NewScoreEvent>();
+
         public void AddNewScore(NewScoreEvent newScoreEvent)
         {
-            int targetBin = (int)Math.Clamp(Math.Floor((float)newScoreEvent.TotalScore / 100000), 0, bin_count - 1);
-            bins[targetBin] += 1;
-            updateCounts();
+            newScores.Enqueue(newScoreEvent);
 
-            var text = new OsuSpriteText
+            // ensure things don't get too out-of-hand.
+            if (newScores.Count > 25)
```

Comment:
I do think this can likely be increased, but keeping it in sync with event feed for now.